src/scripts/utils/inject.ts

```diff
@@ -0,0 +1,44 @@
+const head: HTMLHeadElement = document.head
+const noop = () => {
+}
+
+type ScriptElementConfig = {
+	[P in Exclude<keyof HTMLScriptElement, keyof HTMLElement>]?: HTMLScriptElement[P]
+}
+type ScriptConfig = ScriptElementConfig & { content: string }
+
+
+export default (
+	options: ScriptConfig | string,
```

Comment:
I miss here any hint, that I can pass in a URL string.

Proposing:
- rename to `urlOrOptions` | `optionsOrUrl`
- add type alias `type URLString = string` and then use `ScriptConfig | URLString`